homework3_UPD1/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "homework3",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

```suggestion
    "start": "",
    "test": "echo \"Error: no test specified\" && exit 1"
```
Надо добавлять start - точку входа